.github/workflows/main.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
-        node: [12, 14]
+        node: [14]
```

Comment:
```suggestion
        node: [14, 15]
```

Reply:
15 is blocked by https://github.com/libp2p/js-libp2p/issues/857